src/SonarQube.Common/MutexWrapper.cs

```diff
@@ -35,6 +37,11 @@ public MutexWrapper(string name)
         public MutexWrapper(string name, TimeSpan acquireTimeout)
         {
             mutex = new Mutex(false, name);
+            var allowEveryoneRule = new MutexAccessRule(new SecurityIdentifier(WellKnownSidType.WorldSid, null),
+                MutexRights.FullControl, AccessControlType.Allow);
+            var mutexSecurity = new MutexSecurity();
+            mutexSecurity.AddAccessRule(allowEveryoneRule);
+            mutex.SetAccessControl(mutexSecurity);
```

Comment:
I think there is a race condition here; we don't know whether the current user created the mutex or not, and if they didn't they might not have permissions to set the access control at this point.
There is a constructor that allows the mutex security to be set on creation. I think we should use that instead.

Reply:
Fine by me